project-fortis-pipeline/docs/development-setup.md

```diff
@@ -113,6 +113,14 @@ Note that any changes to the React code in project-fortis-interfaces folder will
 be automatically detected and re-loaded so the re-build step above won't be
 necessary for changes to the frontend.
 
+## Cleaning
```

Comment:
The developer FAQ has moved. To prevent re-work due to merges, I've updated the new location with this snippet, so you can remove this from your PR: https://github.com/CatalystCode/project-fortis/commit/0f2d35181104548e1e46ff0f0d1c0164623de2c5

Reply:
Done in https://github.com/CatalystCode/project-fortis/pull/31/commits/80c7ffa81afaad246c64a19ac40bd012d44de266